ONSdigital/ras-frontstage
# RAS-1305 Introduce non root application user

# What and why?
Adds an app specific user so we don't have it run on the root user - no changes to the image as the one we are currently using has the least number of vulnerabilities
# How to test?
- deploy this PR to your env
- run tests and make sure frontstage still works
- `kubectl exec -it <pod-name>` into the pod and run `ps aux` to make sure that the new `appuser` is running the app instead of the root user
# Jira
[RAS-1305](https://jira.ons.gov.uk/browse/RAS-1305)
